app/Resources/views/admin/blog/show.html.twig

```diff
@@ -4,30 +4,13 @@
 
 {% block main %}
     <h1>{{ post.title }}</h1>
+    <p class="text-muted">{{ post.publishedAt|localizeddate('long', 'medium', null, 'UTC') }}</p>
```

Comment:
Maybe this could be displayed in blog index/show as well? https://github.com/symfony/symfony-demo/issues/105#issuecomment-271021837